crates/breez-sdk/core/src/sdk.rs

```diff
@@ -362,6 +362,17 @@ impl BreezSdk {
                     error!("Failed to sync wallet: {e:?}");
                 }
             }
+            WalletEvent::FoundClaimableTransfer(transfer) => {
+                info!("Found a claimable transfer");
+                if let Ok(payment) = transfer.try_into() {
+                    self.event_emitter
+                        .emit(&SdkEvent::PaymentPending { payment })
```

Comment:
There is a small issue here I think that when app receives this event probably the logic would be to refresh payments list and then the pending payment will not be there since we are not synced yet. Should we add it to the db as pending at this point? If so we also need to make sure the index for next payments page stays correct.